runtime/src/chpl-privatization.c

```diff
@@ -71,8 +72,22 @@ void chpl_newPrivatizedClass(void* v, int64_t pid) {
 }
 
 
-extern void* chpl_getPrivatizedClass(int64_t i) {
+void* chpl_getPrivatizedClass(int64_t i) {
   return chpl_privateObjects[i];
 }
 
 
+void chpl_clearPrivatizedClass(int64_t i) {
+  chpl_privateObjects[i] = NULL;
```

Comment:
Ok. I'd still be inlined to add a short test, just because of how long it took us to discover our last runtime privatization bug